src/DonationForms/resources/registrars/templates/fields/Amount/CustomAmount.tsx

```diff
@@ -22,13 +22,15 @@ const decimalSeparator = formatter.format(1.1).replace(/[0-9]/g, '');
 /**
  * @since 3.0.0
  */
-const CustomAmount = (
-    {defaultValue, fieldError, currency, value, onValueChange}: CustomAmountProps
-) => {
-
+const CustomAmount = ({defaultValue, fieldError, currency, value, onValueChange}: CustomAmountProps) => {
     return (
         <div className={classNames('givewp-fields-amount__input-container', {invalid: fieldError})}>
             <CurrencyInput
+                ref={(input) => {
+                    if (input && fieldError) {
+                        input.focus();
```

Comment:
hmm, i'm pretty sure react-hook-form handles this for us or at least provides a way of doing this for custom inputs.  I know it works properly for other fields since we're using [shouldFocusOnError](https://react-hook-form.com/docs/useform#shouldFocusError) by default. Might want to check the discussion forums https://github.com/orgs/react-hook-form/discussions?discussions_q=is%3Aopen+shouldFocusError

Reply:
bd987a2fd1359c33052e16e32283390db4d71cc2
@jonwaldstein, the focus hooks and settings for react-hook-form all require the ref of the input to be registered. But since this field is registering a hidden input the focus states are being sent to the hidden field rather then the custom amount. This commit simply removes the hidden inputs so that the ref and name can be placed on the custom amount which will allow the focus to follow the ref properly.

https://github.com/user-attachments/assets/7686003d-6ca4-47d7-9ba1-032461bffc5d